### PR TITLE
Updated example for latest WebRx Router changes

### PR DESCRIPTION
**Pull request by oliverw**

### CODE REVIEW

**Comment by samccone** [comment]
:+1:

File: examples/webrx/node_modules/todomvc-common/base.js [hunk 1]
```diff
@@ -114,7 +114,12 @@
 	})({});
 
 	if (location.hostname === 'todomvc.com') {
-		window._gaq = [['_setAccount','UA-31081062-1'],['_trackPageview']];(function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];g.src='//www.google-analytics.com/ga.js';s.parentNode.insertBefore(g,s)}(document,'script'));
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (samccone) [inline examples/webrx/node_modules/todomvc-common/base.js]: analytics?

**Comment by oliverw** [comment]
@samccone Not added by myself for sure.

File: examples/webrx/node_modules/todomvc-common/base.js (hunk 1 above)
Comment (arthurvr) [inline examples/webrx/node_modules/todomvc-common/base.js]: this is expected: https://github.com/tastejs/todomvc-common/commit/6ee744cc2a737541ea80dc8b822b5f87b25ebcb8 ;)

Comment (samccone) [inline examples/webrx/node_modules/todomvc-common/base.js]: kk I was just confused :star:

Comment (arthurvr) [inline examples/webrx/node_modules/todomvc-common/base.js]: That said this does need to be in its own commit.

**Comment by arthurvr** [comment]
Thanks for your PR. Looking good except that we want unrelated changes (updating todomvc-common)  in its own commit :)

**Comment by samccone** [comment]
moved to 1302 thanks again
